### ISSUE TITLE
[FEATURE]  Classify expeditions (the marker) in color by the coral condition

### ISSUE THREAD

**Issue by WWprado** [EPFL-ENAC/lgb-trsc#8]
The coral condition is calculated as the sum of the mean values of coral_alive (subtracte_1).
The higher the result, the better the "coral healthiness".

**Comment by Guilhembp** [comment]
Precisely - what I thought is that you can have a color scheme that shows the sampling sites per expedition (blue for 2022 and yellow for 2023 for example), but you can select a different colour scheme and have them sorted by living coral percent cover, just like I did on QGIS: 

<img width="1516" alt="Image" src="https://github.com/user-attachments/assets/7e1041db-3ea7-44ed-983b-d01b4f5ff183" />

see similar type of presentation with different legends to tick: https://apps.aims.gov.au/reef-monitoring/reefs

The different classes are (taken from https://apps.aims.gov.au/reef-monitoring/reefs): 
0 - 10%: red
10 - 30%: orange
30 - 50%: yellow
50 - 75%: light green
75 - 100%: bright green

**Comment by guilbep** [comment]
@charlottegiseleweil Let's review. I think it's done

<img width="1302" alt="Image" src="https://github.com/user-attachments/assets/91c6ba67-7fd1-4b85-81c1-3431c8867c02" />

**Comment by charlottegiseleweil** [comment]
Yes - all good il faut juste que le point jaune (Marker du pays) disparaisse quand on est dans la scope du pays (sinon c'est confusing avec les points d'exépdition)
